packages/app/src/cli/models/extensions/specifications/app_config_webhook_subscription.ts

```diff
@@ -0,0 +1,78 @@
+import {WebhookSimplifyConfig} from './app_config_webhook.js'
+import {WebhooksConfig} from './types/app_config_webhook.js'
+import {WebhooksSchema} from './app_config_webhook_schemas/webhooks_schema.js'
+import {CustomTransformationConfig, createConfigExtensionSpecification} from '../specification.js'
+import {getPathValue} from '@shopify/cli-kit/common/object'
+
+export const WebhookSubscriptionSpecIdentifier = 'webhook_subscription'
+
+type TransformedWebhookSubscription = {
```

Comment:
I think this name is a bit misleading. It implies that this is the type of the object after you run a transform function, but based on the code, it looks like it's the type of the config in `transformToWebhookSubscriptionConfig` before it's transformed?

Reply:
What do you think about `RemoteWebhookSubscriptionType` or `RemoteWebhookSubscription`? The `reverseTransform` function is called when we are [building the remote app configuration](https://github.com/Shopify/cli/blob/main/packages/app/src/cli/services/app/select-app.ts#L44).